simulation/CMakeLists.txt

```diff
@@ -38,8 +43,13 @@ include_directories("${CMAKE_CURRENT_SOURCE_DIR}/include")
 
 PCL_ADD_LIBRARY(${LIB_NAME} COMPONENT ${SUBSYS_NAME} SOURCES ${srcs} ${incs})
 
+if(PCL_SHARED_LIBS)
+  set(_glew_target GLEW::glew)
```

Comment:
These targets doesn't exist before Cmake 1.15.
https://cmake.org/cmake/help/v3.15/module/FindGLEW.html

Before that, only GLEW::GLEW is defined.